autoconfigure/storage-applicationinsights/src/main/resources/zipkin-server-applicationinsights.yml

```diff
@@ -0,0 +1,7 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=applicationinsights)
+zipkin:
+  storage:
+    applicationinsights:
+      instrumentationKey: ${AI_INSTRUMENTATION_KEY:}
+      applicationId: ${AI_APPLICATION_ID:}
+      apiKey: ${AI_API_KEY:}
```

Comment:
how fast are you running to the query limit? Using Azure Active Directory authentication may eliminate this problem, but may be hard to implement. Just a thought in case you have an idea how to fix it